prod/photo-and-video.section

```diff
@@ -24,6 +24,5 @@ mkvtoolnix-jz
 obs-studio
 smartscreen
 livetuner
-# orphans
 movian
 guvcview
```

Comment:
snap I maintain, but can we add darktable and digikam here?

Reply:
Done!